coss/opensource_clubs/models.py

```diff
@@ -129,3 +129,16 @@ class FAQPage(Page):
     content_panels = Page.content_panels + [
         StreamFieldPanel('faq'),
     ]
+
+
+class ActivitiesPage(Page):
+    """Activities Page for Open Source Clubs."""
```

Comment:
Doesn't this need a title?

Reply:
Nope. Since it inherits from `Page` it already has a title field.